### PR TITLE
Fix/777

### PR DESCRIPTION
**Pull request by elmbeech**
At doc/dev/getting-started.rst I added information for OS X mac port distro users.

### CODE REVIEW

**Comment by elmbeech** [comment]
At doc/dev/getting-started.rst I added information for OS X homebrew distro users.

**Comment by elmbeech** [comment]
At doc/dev/getting-started.rst I added information for OS X homebrew distro user.

**Comment by elmbeech** [comment]
At doc/dev/getting-started.rst I added information for OS X homebrew distro users.

**Comment by mr-c** [comment]
Jenkins, add to test list

**Comment by mr-c** [comment]
Jenkins, ok to test

File: doc/dev/getting-started.rst [hunk 1]
```diff
@@ -95,9 +95,18 @@ One-time Preparation
...
+       sudo port install py27-pip
+     
+   `Homebrew <http://brew.sh/>`__ users on the OS X platform will have pip
+   already installed
+
+     
    `Conda <https://github.com/conda/conda>`__ users on any platform
    should instead create a separate Conda environment::
-
+
```
Comment (mr-c) [inline doc/dev/getting-started.rst]: Extra tab?

File: doc/dev/getting-started.rst [hunk 2]
```diff
@@ -116,6 +125,18 @@ One-time Preparation
...
+   
+   `Mac ports <https://www.macports.org/>`__ users on the OS X platform can
+   install cppcheck by executing from the command line::
+     
+       sudo port install cppcheck
+
+   `Homebrew <http://brew.sh/>`__ users on the OS X platform can
+   install cppcheck by executing from the command line::
+     
+       sudo brew install cppcheck
```
Comment (mr-c) [inline doc/dev/getting-started.rst]: Lets include Ubuntu & Debian Linux: `sudo apt-get install cppcheck`

**Comment by elmbeech** [comment]
- [ ] Is it mergeable?
- [ ] Did it pass the tests?
- [ ] If it introduces new functionality in scripts/ is it tested?
  Check for code coverage with `make clean diff-cover`
- [ ] Is it well formatted? Look at `make pep8`, `make diff_pylint_report`,
  `make cppcheck`, and `make doc` output. Use `make format` and manual
  fixing as needed.
- [ ] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [ ] Is it documented in the ChangeLog?
  http://en.wikipedia.org/wiki/Changelog#Format
- [ ] Was a spellchecker run on the source code and documentation after
  changes were made?
- [ ] Is the Copyright year up to date?

**Comment by elmbeech** [comment]
There is no copyright notification in the getting-started.rst file.
Should I add one?

**Comment by mr-c** [comment]
+1 to copyright header being added; good catch!

**Comment by elmbeech** [comment]
@ mr-c: shoud I add a copy right header to getting-started.rst  or do you prefer to add copyright headers to all the rst files at the same time?

**Comment by mr-c** [comment]
For the purposes of this pull request lets keep it to just this file. We can open an issue for the other files.

**Comment by elmbeech** [comment]
hi Mikael, this bugfix branch should now be ready to get merged with ged master.

**Pull request merged**